packages/cli/src/controllers/auth.controller.ts

```diff
@@ -137,7 +136,7 @@ export class AuthController {
 		}
 
 		try {
-			user = await this.userRepository.findOneOrFail({
+			user = await this.userService.findOneOrFail({
 				relations: ['globalRole'],
```

Comment:
Actually now that I saw that this is always added as part of the service, then maybe we could remove the `globalRole` join